### PR TITLE
Issue 3 : Helm charts for bookkeeper deployment

### LINKED ISSUE #3: Create charts to install bookkeeper operator and bookkeeper cluster
### Description

Currently manual deployment of bookkeeper operator and bookkeeper cluster works. However creating the charts would make the deployment easier.

### Importance

must-have

### Location

Inside charts directory

### Suggestions for an improvement

_(How do you suggest to fix or proceed with this issue?)_

### PR DESCRIPTION
**Pull request by SrishT**
### Change log description
Having helm charts for installation of Bookkeeper Operator and Bookkeeper would make their deployment on Kubernetes easier.

### Purpose of the change
Fixes #3

### What the code does
It deploys Bookkeeper Operator and Bookkeeper on our Kubernetes cluster using helm charts.

### How to verify it
Installation of bookkeeper operator and bookkeeper using charts should be successful.

### CODE REVIEW

File: charts/bookkeeper-operator/values.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # Declare variables to be passed into your templates.
 
 image:
-  repository: pravega/pravega-operator
-  tag: 0.4.0
+  repository: srishtithakkar/bookkeeper-operator
```
Comment (pbelgundi) [inline charts/bookkeeper-operator/values.yaml]: Its fine to use your private repo for testing, but here we should have "pravega/bookkeeper-operator" and tag should be "latest"

Comment (SrishT) [reply charts/bookkeeper-operator/values.yaml]: done

File: charts/bookkeeper/Chart.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 name: pravega
```
Comment (pbelgundi) [inline charts/bookkeeper/Chart.yaml]: Why is this pravega? This is a bookkeper chart yaml... please provide a suitable name.

File: charts/bookkeeper/Chart.yaml [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-version: 0.5.0
-appVersion: 0.5.0
-description: pravega Helm chart for Kubernetes
+version: 0.6.1
+appVersion: 0.6.1
+description: Bookkeeper Helm chart for Kubernetes
 keywords:
 - pravega
 - storage
 home: https://github.com/pravega/pravega
```
Comment (pbelgundi) [inline charts/bookkeeper/Chart.yaml]: This and "sources" perhaps needs to change.

File: charts/bookkeeper/values.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+apiVersion: "bookkeeper.pravega.io/v1alpha1"
+kind: "BookkeeperCluster"
+metadata:
+  name: "pravega"
```
Comment (pbelgundi) [inline charts/bookkeeper/values.yaml]: This is not a pravega cluster. Please use a suitable name ... perhaps "pravega-bk"

Comment (SrishT) [reply charts/bookkeeper/values.yaml]: done

File: charts/bookkeeper/Chart.yaml (hunk 2 above)
Comment (SrishT) [reply charts/bookkeeper/Chart.yaml]: done

File: charts/bookkeeper/Chart.yaml (hunk 3 above)
Comment (SrishT) [reply charts/bookkeeper/Chart.yaml]: done

**Pull request merged**